checkers/discover/generate.go

```diff
@@ -65,3 +65,95 @@ func GenerateAnalyzer(checkerDir, dest string) error {
 
 	return nil
 }
+
+// for file checkers/registry.go
+// modify this template when adding checkers for a new language
+var builtinCheckersGo = `// AUTOMATICALLY GENERATED: DO NOT EDIT
+
+package checkers
+
+import (
+	"globstar.dev/checkers/javascript"
+	"globstar.dev/checkers/python"
+	goAnalysis "globstar.dev/analysis"
+)
+
+type Analyzer struct {
+	TestDir   string
+	Analyzers []*goAnalysis.Analyzer
+}
+
+var AnalyzerRegistry = []Analyzer{
+	{
+		TestDir:   "checkers/javascript/testdata", // relative to the repository root
+		Analyzers: []*goAnalysis.Analyzer{%[1]s
+		},
+	},
+	{
+		TestDir: "checkers/python/testdata",
+		Analyzers: []*goAnalysis.Analyzer{%[2]s
+		},
```

Comment:
There can be more than two `Analyzer{}`s inside the registry. Ideally, the TestDir should also be automatically discovered (we can have a convention for test files). But for now, you can accept the test directory and the analyzer directory from the command line.